azure-pipelines.yml

```diff
@@ -0,0 +1,68 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+
+pr:
+  branches:
+    include:
+      - main
+
+schedules:
+  - cron: '0 0 * * 0'
+    displayName: Weekly build
+    branches:
+      include:
+        - main
+    always: true
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: osisoft/OSI-Samples
+      endpoint: osisoft/OSI-Samples
+
+variables:
+  - template: '/miscellaneous/build_templates/variables.yml@templates'
+  - name: CredFile
+    value: config.yml
+  - name: CredFileLibrary
+    value: product-readiness.data_views_r.r
+  - name: ComparisonFile
+    value: sampleoutput.csv
+  - name: ComparisonFileLibrary
+    value: product-readiness.ocs_data_views_r.output_comparison.csv
+  - name: analysisProject
+    value: Data_Views_R
+
+jobs:
+  - job: Tests
+    pool:
+      name: 00-OSIManaged-Test
+      demands: COMPUTERNAME -equals $(buildAgent)
+    steps:
+      - template: '/miscellaneous/build_templates/config.yml@templates'
+        parameters:
+          fileOriginal: $(CredFile)
+          fileLibrary: $(CredFileLibrary)
+
+      - template: '/miscellaneous/build_templates/config.yml@templates'
+        parameters:
+          fileOriginal: $(ComparisonFile)
```

Comment:
Is this different from the sampleoutput_placeholder.csv that is checked in to the repo? If not, I would just leave it named sampleoutput.csv in the repo, and don't store it in the ADO library at all. It doesn't contain passwords or keys, so I don't think it needs to be kept secret.

Reply:
I was kind of stuck here. Architecture said explicitly to store it outside of the repo, but Beth's comments pointed out that customers running these tests would have a lot of work to get it up and running. 

Using the placeholder style felt better because it's technically following architecture's guidance in that the pipeline is getting its csv from azure, but customer's should hopefully understand what the test is doing if they have manual involvement on the sample. It also crossed my mind that it makes it harder for someone to accidentally commit a csv of sensitive data, but I don't actually see a way for that to happen (unlike a password). 

I'm not really sure what to do for this.

Reply:
Do you know what architecture's reasoning was? We store data in other repos, particularly several sample collections in the bulk upload repos.